src/main/java/com/kit/gui/controller/LoginController.java

```diff
@@ -133,6 +133,7 @@ private boolean loadAccount(String uuid, boolean rememberMe, String email) throw
                         apiKeyProperty.remove();
                     }
                 }
+                Session.get().onAuthenticated();
```

Comment:
Seemed a little pointless because this would get pushed onto the event bus before there's any subscribers